java/arcs/android/crdt/ParcelableRawEntity.kt

```diff
@@ -47,6 +48,7 @@ data class ParcelableRawEntity(
     internal companion object CREATOR : Parcelable.Creator<ParcelableRawEntity> {
         override fun createFromParcel(parcel: Parcel): ParcelableRawEntity {
             val id = requireNotNull(parcel.readString())
+            val expirationTimestamp = requireNotNull(parcel.readString())
```

Comment:
Pretty sure this is going to fail. The string is added to the parcel last, you will need to read it last.

Reply:
argh, i moved one and missed the other, fixed now.